source/central/gds/datagovuk/index.html.md.erb

```diff
@@ -0,0 +1,20 @@
+---
+title: Data.gov.uk
+weight: 31
+---
+
+# Data.gov.uk
+
+**API Base URL links:**
+
+- [Base URL production](https://data.gov.uk/api/action/) - not currently exposed
```

Comment:
What does it mean "not currently exposed"? I can see that `https://data.gov.uk/api/action` returns a 404, but it works as expected as a base url. E.g. `https://data.gov.uk/api/action/package_list` works.

Reply:
I asked why it was returning a 404 and the person who raised the issue said the API was not currently exposed, so I wanted to put a note to that effect. Given that it works as a base URL I'll remove